internal/config/config.go

```diff
@@ -103,3 +122,13 @@ func (c *Config) AssignClients() map[string][]hotstuff.ID {
 	}
 	return hostsToClients
 }
+
+// Returns true if both the replica and client hosts slices
+// contain one instance of "localhost".
+func (c *HostConfig) IsLocal() bool {
+	if len(c.ClientHosts) > 1 || len(c.ReplicaHosts) > 1 {
+		return false
+	}
+
```

Comment:
Nit: I prefer to avoid empty lines in short methods, especially if there is a single `}` (almost empty line) above it.